uievents/order-of-events/mouse-events/click-cancel.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Default action is canceled if the click event is canceled.</title>
+    <link rel="help" href="https://w3c.github.io/uievents/#events-clickevent-event-order">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+  </head>
+  <body>
+    <ol>
+      <li>Check the following item.</li>
+      <li>Hover over the button.</li>
```

Comment:
@gsnedders is this old cruft?

Reply:
I was trying to make a minimal change.

Reply:
https://github.com/w3c/web-platform-tests/pull/8089